.mvn/checksums/checksums-central.sha256

```diff
@@ -0,0 +1,1084 @@
+0670b605255f7dc9a454daaec7912918ccf1b5475cbfca374363b51fcfd4ea00  org/codehaus/plexus/plexus-component-annotations/2.1.0/plexus-component-annotations-2.1.0.pom
+06aa5d1ac068f2f809530f3ff53fb5c8c940152a253623d0b53c4f53f5a13a3d  com/google/guava/guava-parent/32.1.1-jre/guava-parent-32.1.1-jre.pom
+06cfb3e43bb5d417aff2418dbbf023884e644f7233ba841b47602662e1be7f68  org/apache/maven/plugins/maven-wrapper-plugin/3.3.2/maven-wrapper-plugin-3.3.2.jar
+06d75dd6f2a0dc9ea6bf73a67491ba4790f92251c654bf4925511e5e4f48f1df  org/sonatype/sisu/sisu-inject-bean/1.4.2/sisu-inject-bean-1.4.2.pom
+06f3c3ddad57b30bfe88655456a04731e56a78ad0dd909e65c71881003b96479  com/google/inject/guice/4.2.2/guice-4.2.2.pom
+07522240db7dd10dcba8897780c08bc4a00cfe71ff9e35be249695bf4a735601  org/apache/commons/commons-parent/57/commons-parent-57.pom
+079f1ce924197890598b651f515b8337261cb37a0714341ed5ff5b312736b316  org/apache/maven/plugins/maven-surefire-plugin/3.1.2/maven-surefire-plugin-3.1.2.jar
+07b1586faf220c05821d0f3ed8e2e417e214c83f40641f76e8a90b134c31ff6b  ch/qos/logback/logback-core/1.2.13/logback-core-1.2.13.jar
+0859ba0d5cfeefb13964e5862c036faa48a0bccff4932638fb13fe3445df33f7  org/objenesis/objenesis-parent/3.2/objenesis-parent-3.2.pom
+0868e3c558341f627c40130886b30eb816440ecb3083ab3ccce4f8ded0c53276  org/mockito/mockito-scala_2.13/1.17.5/mockito-scala_2.13-1.17.5.jar
+093be1b03331bce2932d6825c37e98272d7621e6a9e9fb93289a002518b8dd5a  org/tukaani/xz/1.9/xz-1.9.pom
+093fea360752de55afcb80cf713403eb1a66cb7dc0d529955b6f4a96f975df5c  commons-collections/commons-collections/3.2/commons-collections-3.2.jar
+0973e876682bc72016919be4569e1b1bfc462429cc4e0c937f54e3cee075c155  com/typesafe/akka/akka-parsing_2.13/10.2.7/akka-parsing_2.13-10.2.7.jar
+0987adf8d5201d5db0dce8deef9d325701eb341e306a624e5a9870fbab5b6da5  org/json4s/json4s-ast_2.13/4.0.3/json4s-ast_2.13-4.0.3.pom
+09b999a969e73525a6cc3ad2868ea744766e1d93b25c6c656d61a5ff9c881da9  org/codehaus/plexus/plexus/1.0.10/plexus-1.0.10.pom
+09c95ce9300bb535bc1c5ec0548b95d44ff9f67509f46a8c713ad10a235e031f  org/scalatest/scalatest_2.13/3.2.16/scalatest_2.13-3.2.16.pom
+09d992752c32b279e152481cc700296f6fc8a7d7395f68eef7a581da51915328  com/typesafe/akka/akka-coordination_2.13/2.6.20/akka-coordination_2.13-2.6.20.jar
+0a1b692d7fcc90d6a45dae2e50f4660d48f7a44504f174aa60ef34fbe1327f6a  org/codehaus/plexus/plexus/4.0/plexus-4.0.pom
+0a57044d10895f8d3dd66ad4286891f607169d948845ac51e17b4c1cf0ab569d  org/scala-lang/scala-library/2.12.10/scala-library-2.12.10.jar
+0a5b2a9a80edf32415e4e231e589fd791db367ebf8d1ad24a308af51be6418a1  org/apache/maven/maven-plugin-api/3.8.6/maven-plugin-api-3.8.6.pom
+0addec670fedcd3f113c5c8091d783280d23f75e3acb841b61a9cdb079376a08  aopalliance/aopalliance/1.0/aopalliance-1.0.jar
+0ae83d4309760efa5bebf9e377a5b3b971f4efe7f39a29513406a46b8af7ee1a014fbf91ae76986a57da2ccafffbb92942de154d47de79a645ad650e143c7b9a  org/apache/maven/plugins/maven-deploy-plugin/3.1.2/maven-deploy-plugin-3.1.2.pom
```

Comment:
That's a SHA512 👀

Reply:
fixed in https://github.com/ACINQ/eclair/pull/2998/commits/9aa44ff80d1df60a09572534348a7ac315b7664b